content/scout/metrics-exporter.md

```diff
@@ -0,0 +1,174 @@
+You will need to provide this token to the exporter when scraping metrics.
+
+## Configure Prometheus
+
+### Add a job for your organization
+
+In the Prometheus configuration file, add a new job for your organization.
+The job should include the following configuration;
+replace `ORG` with your organization name:
+
+```yaml
+scrape_configs:
+  - job_name: $ORG
+    honor_timestamps: true
+    metrics_path: /v1/exporter/org/<ORG>
+    scheme: https
+    static_configs:
+      - targets:
+          - api.scout.docker.com
+```
+
+The address in the `targets` field is set to the domain name of the Docker Scout API, `api.scout.docker.com`.
+Make sure that there's no firewall rule in place preventing the server from communicating with this endpoint.
+
+### Scrape interval
+
+By default, Prometheus scrapes the metrics every 15 seconds.
+You can change the scrape interval by setting the `scrape_interval` field in the Prometheus configuration file at the global or job level.
+A scraping interval of 60 minutes or higher is recommended.
+
+Because of the own nature of vulnerability data, the metrics exposed through this API are unlikely to change at a high frequency.
+For this reason, the metrics endpoint has a 60-minute cache by default.
+If you set the scrape interval to less than 60 minutes, you will see the same data in the metrics for multiple scrapes during that time window.
+
+### Add bearer token authentication
+
+To scrape metrics from the Docker Scout Exporter endpoint using Prometheus, you need to configure Prometheus to use the PAT as a bearer token.
+The exporter requires the PAT to be passed in the `Authorization` header of the request.
+
+Update the Prometheus configuration file to include the `authorization` configuration block.
+This block defines the PAT as a bearer token stored in a file:
+
+```yaml
+scrape_configs:
+  - job_name: $ORG
+    authorization:
+      type: Bearer
+      credentials_file: /etc/prometheus/token
+```
+
+The content of the file should be the PAT in plain text:
+
+```console
+dckr_pat_...
+```
+
+If you are running Prometheus in a Docker container or Kubernetes pod, mount the file into the container using a volume or secret.
+
+Finally, restart Prometheus to apply the changes.
+
+## Compose example
+
+If you don't have a Prometheus server set up, you can run a demo using Docker Compose.
+The demo includes a Prometheus server that scrapes metrics for a Docker organization enrolled in Docker Scout,
+alongside Grafana with a pre-configured dashboard to visualize the vulnerability and policy metrics.
+
+1. Download the starter template for bootstrapping a set of Compose services
+   for scraping and visualizing the Docker Scout metrics endpoint:
+
+   <a href="/assets/files/scout-metrics-template.zip" download>Download template</a>
+
+2. Unzip and `cd` into the directory.
+
+   ```console
+   $ unzip scout-metrics-template.zip
+   $ cd scout-metrics-template
+   ```
```

Comment:
I'd rather use a public repo for the demo assets. This way if we need to make a change (like updating a dashboard with more graphs) we won't need to re-upload the .zip file.

Reply:
Sure. I've pinged the devrel team to help me set up a repository for this under the `dockersamples` org. I'll update here when that's all set up

Reply:
Done, I've created a repo and uploaded the files there. It's referred to from this guide now. PTAL (repo here: https://github.com/dockersamples/scout-metrics-exporter)